docker-compose.dev.yml

```diff
@@ -20,6 +20,8 @@ services:
       - SESSION_SECRET=super_secret_session_key_change_me_in_production_please
       - CI=
       - NODE_ENV=development
+      - LOCALSTACK_HOSTNAME=localstack
```

Comment:
I also needed to add this env var to the `console` service to run the test script like `docker compose -f ./docker-compose.dev.yml run --rm -it console yarn rw exec tests3     ` - is that expected? Or should I have run this a different way?

Reply:
ah you're right, will add